### PR TITLE
Edit and shorten Contact Us section

### PR DESCRIPTION
**Pull request by reece**
Implements comments on #194.

### CODE REVIEW

**Comment by reece** [comment]
@marinadistefano @ahwagner @ifokkema https://hgvs-nomenclature--200.org.readthedocs.build/200/ shows the changes from #194:

- merge first two paragraphs under discussion group
- dropped the Q&A link.
- renamed the section and included the announcement list

If you all like the new text regarding response time, keep it. My 2¢: It's unnecessary and not worth the space, and it implies a service expectation that I doubt we're really ready to guarantee. And it addresses a problem that we don't have (thanks primarily to Johan).

File: docs/index.md [hunk 1]
```diff
@@ -43,10 +42,7 @@ expedited response in relation to a clinical case, please indicate this in both
 the subject line and body of the email inquiry. We cannot guarantee that even
 clinical inquiries will be addressed outside of business hours or on weekends.
 
-You may also submit questions to our
```
Comment (jfjlaros) [inline docs/index.md:46]: Why do we want to get rid of this?

Comment (ifokkema) [reply docs/index.md:46]: See #194.

File: docs/index.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ The HGVS Nomenclature is administered by the [HGVS Variant Nomenclature Committe
 [//]: # (    If you are interested in joining the HVNC and contributing to the maintenance of the HGVS Nomenclature, please see the [Call for Members]&#40;call-for-members.md&#41;.)
 [//]: # (    Applications are due **April 1, 2024**.)
 
-## Contact Us
+<a id="contact-us"></a>
```
Comment (reece) [inline docs/index.md:13]: This ensures that any #contact-us anchors still work.

**Comment by reece** [comment]
@marinadistefano @ahwagner @ifokkema @jfjlaros This PR is now ready for review. The paragraph with response expectations has been retained as-is.

**Pull request merged**